internal/commands/build/types.go

```diff
@@ -46,6 +52,13 @@ func loadServices(servicesDict map[string]interface{}, buildArgs []string) ([]Se
 			return nil, err
 		}
 		services = append(services, *serviceConfig)
+
+		// Sanity check
+		for _, volume := range serviceConfig.Volumes {
+			if volume.Type == "bind" && strings.HasPrefix(volume.Source, ".") {
```

Comment:
Relative paths not necessarily start by `.`. 
`./path/to/file` and `path/to/file` are equivalent

Reply:
oh indeed, this is just an habit of mine.